darwin/cli_functions.py

```diff
@@ -257,13 +257,56 @@ def dataset_report(dataset_slug: str, granularity: str) -> None:
     dataset_slug: str
         The dataset's slug.
     granularity: str
-        Granualarity of the report, can be 'day', 'week' or 'month'.
+        Granularity of the report, can be 'day', 'week' or 'month'.
     """
     client: Client = _load_client(offline=True)
+    console = Console(theme=_console_theme())
```

Comment:
I think we should add a `--pretty` argument to this CLI command. When `--pretty` is specified, then the report is printed with rich, otherwise we keep it as it is now

Reply:
Done